nilearn/_utils/cache_mixin.py

```diff
@@ -117,6 +119,10 @@ def cache(func, memory, func_memory_level=None, memory_level=None,
         be cached or not (if user_memory_level is equal of greater than
         func_memory_level the function is cached)
 
+    shelve: boolean,
```

Comment:
No comma needed I think. Also I tend to prefer Python types in general so bool rather than boolean.